.github/workflows/quarkus.yml

```diff
@@ -128,6 +128,12 @@ jobs:
       with:
         name: graalvm
         path: graalvm.tgz
+    # Use Java 17 to build Quarkus as that's the lowest supported JDK version currently
+    - uses: graalvm/setup-graalvm@v1
+      with:
+        version: 'latest'
```

Comment:
`version` is deprecated. As much as I like using `setup-graalvm`, maybe it's better to use `setup-java` instead because here, we only care about compiling with javac?

Reply:
Mostly yes. We decided to use GraalVM just as a very basic test that Quarkus compiles with it.

Reply:
Updated to using Temurin builds of JDK 17

Reply:
Thanks! Could we change this to use Oracle JDK? I can make the change unless you'd like to do it.

Reply:
Sure, please go ahead.